docker/.env

```diff
@@ -51,3 +51,9 @@ SUPERSET_LOAD_EXAMPLES=yes
 CYPRESS_CONFIG=false
 SUPERSET_PORT=8088
 MAPBOX_API_KEY=''
+
+OKTA_BASE_URL=https://dev-82570728.okta.com
```

Comment:
Rename to generic values since this could be any OIDC provider.

Reply:
Hi @jamescrowley321 

i think it should be same like this suppose we have multiple sso provider then we need to add individual names.